TarponSimulator2017/Core/Physic/GameObject.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Xna.Framework;
+using System.Linq;
+
+/// <summary>
+/// Game object.
```

Comment:
We might want to add a link to the video of computerphile : https://www.youtube.com/watch?v=vQ60rFwh2ig